### PR TITLE
Snowmap Salvage Project PR#9 - Saint Corgi

### PR DESCRIPTION
**Pull request by jknpj**
They are a corgi/saint bernanrd mix breed that look for people dying from hypothermia and feed them with their magical leporazine barrel. Even though they have cold resistance they aren't space-proof so keep that in mind! Sprite by @NigglyWiggly

### CODE REVIEW

**Comment by ghost** [comment]
<3

**Comment by SonixApache** [comment]
post sprite(s)

**Comment by jknpj** [comment]
But Icondiffbot did tha- Ok, i'll do that later.

**Comment by ihadtoregisterforthis** [comment]
>the icon bot actually shows the sprites

For how long has this been a thing again?

**Comment by NigglyWiggly** [comment]
image

File: code/modules/mob/living/simple_animal/friendly/corgi.dm [hunk 1]
```diff
@@ -640,3 +640,74 @@
...
+/mob/living/simple_animal/corgi/saint
+	name = "saint corgi"
+	real_name = "saint corgi"
+	desc = "It's a saint bernard corgi mix breed. It has a tiny rescue barrel strapped around his collar to warm up travelers."
+	icon_state = "saint_corgi"
+	icon_living = "saint_corgi"
+	icon_dead = "saint_corgi_dead"
+	health = 60
+	maxHealth = 60
+	minbodytemp = 0
```
Comment (MadmanMartian) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm:657]: Absolute zero corgi?

Comment (Probe1) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:657]: 😆

Comment (jknpj) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:657]: It's a feature, no cold can stop them.

File: code/modules/mob/living/simple_animal/friendly/corgi.dm [hunk 2]
```diff
@@ -640,3 +640,74 @@
...
+	var/turns_since_scan = 0
+	var/mob/living/carbon/victim = null
+	can_breed = FALSE //tfw no gf
```
Comment (Probe1) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm:660]: you're a collaborator you're supposed to be teaching these young coders GOOD practices.

**Comment by Probe1** [comment]
I got like 3/4ths the way through and lost interest.  The start was good but it dragged on and I didn't want to finish.  

Code Review Score:  7/10

File: code/modules/mob/living/simple_animal/friendly/corgi.dm (hunk 2 above)
Comment (jknpj) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:660]: :cry:

Comment (jknpj) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:660]: There's a deep message in that comment filled with hope so the next generation can finally add puppies and a female cogi bernard

**Pull request merged**